scripts/docker-setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
+CHAIN=$(echo $1 | tr '[:lower:]' '[:upper:]')
+
+case $CHAIN in
+  INTR)
+    ;;
+  *)
+    CHAIN=KINT
```

Comment:
nit: `echo` something here to instruct the caller that their flag has either been overridden or that a default was applied